.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.34.0
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.34.0
         with:
-          java-version: '21'
+          java-version: '17'
```

Comment:
Wouldn't removing java version specified default to java 17?

Reply:
Yes, the default version is 17 hence removed java-version tag from here.